components/tools/OmeroPy/src/omero/testlib/__init__.py

```diff
@@ -693,15 +693,13 @@ def create_pixels(self, x=10, y=10, z=10, c=3, t=50, pixels_type="int8",
 
     @classmethod
     def get_pixels_type(cls, pixels_type):
-        if pixels_type in ["int8", "uint8"]:
-            return 1
         if pixels_type in ["int16", "uint16"]:
             return 2
         if pixels_type in ["int32", "uint132", "float"]:
             return 4
         if pixels_type in ["double"]:
             return 8
-        return -1
```

Comment:
Perhaps this should throw? I assume that what would have happened if `-1` got passed to the server.